### ISSUE TITLE
Confirmation message in ad chart interface causes dropdown to widen unpleasantly

### ISSUE THREAD

**Issue by alisman** [cBioPortal/cbioportal#10737]
![image](https://github.com/cBioPortal/cbioportal/assets/186521/8a255b96-7266-4962-a2cc-cdf06bda4a89)

**Comment by mhsh312** [comment]
How about defining a max-width for the confirmation message so that the message box increases in height extra text just goes to the next line?

**Comment by alisman** [comment]
@mhsh312 the menu as a whole needs to be flexible width.   the confirmation message should always fill but never push out the menu.  this should be possible with css.  give it a shot!  thanks!

**Comment by mhsh312** [comment]
@alisman so are you saying that when the message is too big and wants more space, the whole menu should grow out instead of just the message?

**Comment by alisman** [comment]
No, i want the message text to wrap and grow vertically when necessary.  It
should not cause the menu to expand horizontally.

**Comment by mhsh312** [comment]
Ok, makes sense. I will make some css changes to the `SuccessBanner` component and that should fix it.
